Lib/test/test_embed.py

```diff
@@ -1349,44 +1352,32 @@ def test_init_pyvenv_cfg(self):
                 paths[-1] = lib_dynload
             else:
                 for index, path in enumerate(paths):
-                    if index == 0:
-                        paths[index] = os.path.join(tmpdir, os.path.basename(path))
-                    else:
-                        paths[index] = os.path.join(pyvenv_home, os.path.basename(path))
```

Comment:
Does this reflect a change in behavior or just merging two branches that actually did the same thing?

Reply:
A change in behaviour, but only for the test, not for regular venvs. I've fixed both and added a comment explaining why the test is weird.